lms/urls.py

```diff
@@ -20,9 +20,7 @@
 from openedx.features.enterprise_support.api import enterprise_enabled
 
 
-# Uncomment the next two lines to enable the admin:
-if settings.DEBUG or settings.FEATURES.get('ENABLE_DJANGO_ADMIN_SITE'):
-    admin.autodiscover()
+django_autodiscover()
```

Comment:
They should be doing things the same way, with the exception that CMS always allows the admin URLs while LMS doesn't. Is the suggestion to add an LMS feature flag to match?